### PR TITLE
[7565] Update openapi spec to remove unnecessary payload fields

### PR DESCRIPTION
**Pull request by defong**
### Context
Unnecessary payload fields
### Changes proposed in this pull request
Update openapi spec to remove unnecessary payload fields

### Guidance to review
`lead_school_urn` no longer exists and only the docs need updating

### Important business

- [ ] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [ ] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?
- [ ] Do we need to send any updates to DQT as part of the work in this PR?
- [ ] Does this PR need an ADR?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

**Comment by defong** [comment]
[Update OpenAPI spec to remove unnecessary payload fields](https://trello.com/c/WBDTglat/7565-update-openapi-spec-to-remove-unnecessary-payload-fields)

File: app/serializers/api/v0_1/trainee_serializer.rb [hunk 1]
```diff
@@ -25,6 +25,9 @@ class TraineeSerializer
         applying_for_grant
         applying_for_scholarship
         bursary_tier
+        course_max_age
```
Comment (Reyko) [inline app/serializers/api/v0_1/trainee_serializer.rb:28]: Don't we need to return this in the response?

**Review by Reyko — APPROVED** [review]

**Review by tomtrentham — COMMENTED** [review]
I'm not sure this has removed the specified fields from the payload in the docs.
image

**Comment by defong** [comment]
Discussed with @tomtrentham ambiguous ticket led to assumed expectation back to the drawing board.

**Pull request closed without merging**